koku/cost_models/serializers.py

```diff
@@ -289,6 +289,10 @@ def source_type_internal_value_map(self):
     def validate(self, data):
         """Validate that the source type is acceptable."""
         # The cost model has markup, no rates, and is for a valid non-OpenShift source type
+        source_type = data.get('source_type', '')
```

Comment:
why does this get set to `''` and then we check `if source_type` on the next line?
Is there a case where this data can be `None` if so should we be throwing a validation error?

Reply:
This was a mistake! I think I had it set to `''` before I checked for the `source type` and forgot to remove it. Good catch! - Fixed 👍